lib/active_fedora/associations/builder/directly_contains.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveFedora::Associations::Builder
+  class DirectlyContains < CollectionAssociation #:nodoc:
+    self.macro = :directly_contains
+    self.valid_options += [:has_member_relation, :is_member_of_relation]
+    self.valid_options -= [:predicate]
+
+    def build
+      reflection = super
+      configure_dependency
+      reflection
+    end
+
+    def validate_options
+      super
+      if !options[:has_member_relation] && !options[:is_member_of_relation]
+        raise ArgumentError, "You must specify a predicate for #{name}"
```

Comment:
Oh, yes, it is in the superclass: https://github.com/projecthydra/active_fedora/blob/master/lib/active_fedora/associations/builder/association.rb#L9